website/docs/releases/release-notes.md

```diff
@@ -2,6 +2,21 @@
 title: Release Notes
 ---
 
+## v0.7.0-next.0
+#### cashc compiler
+- :sparkles: Add destructuring assignments, e.g. `bytes2 left, bytes1 right = 0x123456.split(2)`
+- :sparkles: Add multiplication, e.g. `int x = 5 * 5`
+- :sparkles: Add *native* introspection/covenants
+- :boom: **BREAKING**: Remove all old introspection/covenant functionality (`tx.version`, `tx.hashPrevouts`, `tx.hashSequence`, `tx.outpoint`, `tx.bytecode`, `tx.value`, `tx.sequence`, `tx.hashOutputs`, `tx.locktime`, `tx.hashtype`, `OutputP2PKH`, `OutputP2SH`, `OutputNullData`)
+  - See the [migration notes](/docs/releases/migration-notes#v06-to-v07) for details on migrating from the old introspection to the new *native* inntrospection methods.
+- :boom: **BREAKING**: Remove `sig` to `datasig` casting
```

Comment:
Possible to add a short note about what devs should do instead?

Reply:
Added to the migration notes!